eligibility_verification/core/migrations/0001_initial.py

```diff
@@ -22,23 +22,31 @@ class Migration(migrations.Migration):
             name='EligibilityVerifier',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('name', models.CharField(max_length=50)),
+                ('name', models.CharField(max_length=25)),
                 ('api_url', models.CharField(max_length=50)),
+                ('api_auth_header', models.CharField(max_length=50)),
+                ('api_auth_key', models.CharField(max_length=100)),
+                ('public_key_pem', models.TextField()),
+                ('jwe_cek_enc', models.CharField(max_length=25)),
+                ('jwe_encryption_alg', models.CharField(max_length=25)),
+                ('jws_signing_alg', models.CharField(max_length=25)),
                 ('eligibility_types', models.ManyToManyField(to='core.EligibilityType')),
             ],
         ),
         migrations.CreateModel(
             name='TransitAgency',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('slug', models.CharField(max_length=5)),
```

Comment:
Save yourself some annoyance: Any time you need a "string" field in sql, just use `TEXT` type.

Reply:
These are Django fields, part of how their ORM layer - but fair point, they could all be Django `models.TextField`.

Reply:
Thanks for the suggestions, I went ahead and changed all these fields to Django `model.TextField` between this PR and #4.